docs/specs/servers/notify/notify-server-api.md

```diff
@@ -2,183 +2,244 @@
 
 ## Authentication
 
-All endpoints expect an `Authorization` header in the form `Authorization: Bearer <project_secret>` using the project secret associated with a project ID. The secret used should be the one that was generated automatically when configuring notify - with the name `notify_api_secret`.
+All endpoints expect an `Authorization` header in the form `Authorization: Bearer <notify_api_secret>` using the notify_api_secret associated with a project ID. The secret used should be the one that was generated automatically when configuring notify - with the name `notify_api_secret`.
 
-## Notify
+## Send Notifications
 
-Send a notification to a set of accounts.
+Send 1 or more notifications to accounts. Notifications handled itempotently are deduplicated by `notificationId`.
 
-`POST /notify`
-
-Body:
+`POST /v1/<project-id>/notify`
 
 ```typescript
-{
-  "notification": {
-    "title": string,
-    "body": string,
-    "icon": string,
-    "url": string,
-    "type": string,
+type Body = [{
+  // Idempotency key and ID to track status
+  notificationId?: string | null,
```

Comment:
since notification id is on the level of the entire request, where one notification could include many users,, how would we account to different success statuses amongst users?

Say notif A was sent to users 1,2,3. 1 had successful message delivery, 2 was not subscribed at all, 3 was not subscribed to the notification type.

What would the status of that notification be? how would users of this API who to resend to?

Reply:
See the endpoint below for your answer. The status isn't per notification, but per account for a notification.